README.md

```diff
@@ -14,61 +14,148 @@ At Pensano Developers, we are committed to giving back to our community and supp
 
-  ```bash
-  cd backend
-  npm install
-  cd ../frontend
-  npm install
-  ```
-3. Install the "react-allergens" package using the package manager of your choice, such as npm or yarn.
-
-To install "react-allergens" using npm:
-  ```bash
-  npm install react-allergens
-  ```
+```bash
+cd backend
+npm install
+cd ../frontend
+npm install
+```
+
+3. Ensure nodemon is installed by running
+
+```bash
+npm install nodemon --global
+```
+
+4. Install the "react-allergens" package using the package manager of your choice, such as npm or yarn.
+   To install "react-allergens" using npm:
+
+```bash
+npm install react-allergens
+```
+
 To install "react-allergens" using yarn:
-  ```bash
-  yarn add react-allergens
-  ```
-4. Get a ChatGPT API key - you will need it later. If your team does not have one, you will have to sign in to your ChatGPT account, go to the API keys page and click the “Create new secret key” button. This key should not be visible on GitHub, so we will use a .env file to keep it hidden. Publicly exposing your key can result in your account being compromised, which could lead to unexpected charges on your account.   
 
-5. Confirm that `.env` has been added to your `.gitignore` dir
+```bash
+yarn add react-allergens
+```
+
+5. Get a ChatGPT API key - you will need it later. If your team does not have one, you will have to sign in to your ChatGPT account, go to the API keys page and click the “Create new secret key” button. This key should not be visible on GitHub, so we will use a .env file to keep it hidden. Publicly exposing your key can result in your account being compromised, which could lead to unexpected charges on your account.
+
+6. Confirm that `.env` has been added to your `.gitignore` dir
+
+7. Install dotenv:
 
-6. Install dotenv:
 ```bash
   npm install dotenv --save
-  ```
-7. Create a `.env` file in the backend dir and add `OPENAI_API_KEY=` followed by your ChatGPT API key. It should look like this:
-  ```
-  PROJECT_API_KEY=3ebf52c0-3ebf52c0
-  ```
+```
+
+8. Create a `.env` file in the backend dir and add `OPENAI_API_KEY=` followed by your ChatGPT API key. It should look like this:
+
+```
+PROJECT_API_KEY=3ebf52c0-3ebf52c0
+```
 
 ### Running
+
 1.  To start the frontend and backend servers at the same time:
 
-  ```bash
-  cd backend
-  npm start
-  ```
+```bash
+cd backend
+npm start
+```
 
 2. To start the frontend only:
 
-  ```bash
-  cd frontend
-  npm start
-  ```
+```bash
+cd frontend
+npm start
+```
 
 3. To start the backend only:
 
-  ```bash
-  cd backend
-  npm run server
-  ```
+```bash
+cd backend
+npm run server
+```
 
 ### Resources
+
 - [MERN Stack: A Comprehensive Guide](https://blog.nextideatech.com/how-to-get-started-with-the-mern-stack-a-comprehensive-guide/)
 - [Keeping your API keys secret with dotenv](https://jonathansoma.com/lede/foundations-2019/classes/apis/keeping-api-keys-secret/)
 - [dotenv](https://www.npmjs.com/package/dotenv)
+
+# Getting Started with Create React App
```

Comment:
I'm all in favour of ditching the default React App documentation as it feels like unnecessary information for this project. If we are keeping it then line 98, "In the project directory, you can run:" is no longer true and line 100, "### `npm start`" is repetition.

Reply:
Agreed. Another thing we added yesterday was to set our ‘OPENAI_API_KEY’ Environment Variable using zsh:

1. Run the following command in your terminal, replacing yourkey with your [API key](https://beta.openai.com/account/api-keys). 

echo "export OPENAI_API_KEY='yourkey'" >> ~/.zshrc
 

2. Update the shell with the new variable:

source ~/.zshrc
 

3. Confirm that you have set your environment variable using the following command. 

echo $OPENAI_API_KEY
The value of your API key will be the resulting output.

Reply:
I will make the changes and let you know.